### PR TITLE
feat: Add next.js + strictest

### PR DESCRIPTION
**Pull request by akira-toriyama**
Added `tsconfig.json` for Next.js (strictest.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -118,6 +118,22 @@ Add to your `tsconfig.json`:
 ```json
 "extends": "@tsconfig/next/tsconfig.json"
 ```
+
+### Next.js + Strictest <kbd><a href="./bases/next-strictest.json">tsconfig.json</a></kbd>
```
Comment (akira-toriyama) [inline README.md:122]: ---

https://nextjs.org/docs/basic-features/typescript

> TypeScript strict mode is turned off by default. When you feel comfortable with TypeScript, it's recommended to turn it on in your tsconfig.json.

**Comment by orta** [comment]
Looks good, CI wants you to run `deno run --allow-read --allow-write scripts/update-markdown-readme.ts` to update the README

**Comment by akira-toriyama** [comment]
@orta 

Thank you.
I did not read it carefully 😱

https://github.com/tsconfig/bases#contributing

**Comment by orta** [comment]
It happens! Thanks, this should get deployed over night

**Pull request merged**